packages/tocco-ui/src/SomeOf/SomeOf.spec.js

```diff
@@ -0,0 +1,17 @@
+import {mount} from 'enzyme'
+import React from 'react'
+
+import SomeOf from './SomeOf'
+
+describe('tocco-ui', () => {
+  describe('SomeOf', () => {
+    test('should display values', () => {
+      const wrapper = mount(<SomeOf some={12} of={345}/>)
+      expect(wrapper.find('span').text()).to.equal('12 / 345')
```

Comment:
in general it would be the better approach to check whether the output contains the some and of value then to check for a very specific format.
e.g. we could replace "/" with a ":" and the test should not fail IMO

Reply:
https://www.chaijs.com/plugins/chai-string/ or https://www.chaijs.com/plugins/chai-match/ would be helpful. Please consider to add one of those.